slides/src/content/a11y.md

```diff
@@ -215,6 +217,36 @@ For example in your header div:
 
 ---
 <!-- .slide: id="a11y" -->
+## Accessible Development - Defining Page Sections
+###CSS Clip Attribute
+
+- Rather than use an offscreen-text class, CSS offers a `clip:` attribute natively that can be used to set text to an 'invisible' state, which you can use similarly
+- The advantage of using `clip:` attribute is that the text is truly not visible to non screen-reader users.  Off screen text may become visible on a large screen in a responsive site
+- The basic syntax is `clip: { shape | auto | inherit } ;` and it only works with absolute or fixed position, not relative
+  - shape - only the `rect()` shape exists
+  - auto - sets the default behaviour at full image / text size - like not using clip at all
+  - inherit - will inherit the clip from its parent
+
+---
+<!-- .slide: id="a11y" -->
+## Accessible Development - Defining Page Sections
+###CSS Clip Attribute Example
+```
+<style>
+.clp {
+    position: absolute; 
+    clip: rect(1px,1px,1px,1px); 
+}
+</style>
+</head>
+<body>
+
+<h1 class=".clp">This is offscreen text</h1>
+</body>
+```
+
+---
+<!-- .slide: id="a11y" -->
```

Comment:
see my comments on issue 530 - I've created a separate skip to nav example.- how it plays out with voice over (VO)